util/src/main/scala/io/catbird/util/var.scala

```diff
@@ -39,15 +38,9 @@ trait VarInstances1 {
 }
 
 private[util] abstract class VarCoflatMap extends CoflatMap[Var] {
+
   final def coflatMap[A, B](fa: Var[A])(f: Var[A] => B): Var[B] = Var(f(fa))
 
-  /**
-   * Note that this implementation is not stack-safe.
-   */
-  final def tailRecM[A, B](a: A)(f: A => Var[Either[A, B]]): Var[B] = f(a).flatMap {
```

Comment:
It looks like the law-checking for `Var` is still using `StackUnsafeMonad`, though?

Reply:
Yes, it still is. The easy fix of using `StackSafeMonad` failed and rather than leave the code seeming to be be stack safe I left it unchanged (and still commented). I didn't have the time at hand to dive into `Var` to understand why it's not behaving.